src/main/java/core/basesyntax/service/impl/ParserImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.Parser;
+
+public class ParserImpl implements Parser {
+    public static final ValidatorImpl validatorImpl = new ValidatorImpl();
+    

```

Comment:
Тут не соблюдается принцип DIP
```suggestion
public class ParserImpl implements Parser {
    public static final Validator validator;
    
    public ParserImpl(Validator validator) {
    this.validator = validator;
    }

```